lib/config/presets/internal/monorepo.ts

```diff
@@ -184,6 +184,7 @@ const repoGroups = {
   uppy: 'https://github.com/transloadit/uppy',
   vue: 'https://github.com/vuejs/vue',
   vuepress: 'https://github.com/vuejs/vuepress',
+  wcf: 'https://github.com/dotnet/wcf',
```

Comment:
```suggestion
  'dotnet-wcf': 'https://github.com/dotnet/wcf',
```
Sounds better to me

Reply:
I don't think that's necessary since there isn't (to my knowledge) another WCF project to confuse this one with, and from a brief review of the full contents of this file, we don't prefix most other monorepos with the organization/platform name (e.g. we have `vstest` rather than `microsoft-vstest`). I can certainly change this if you want - it just seems inconsistent with the style already established.

Reply:
Here the dotnet distinguishes between languages, e.g. in case someone creates a `web css factory` in JS

Reply:
OK. I'll push another change to fix the file's alphabetization.